components/Text/Text.module.scss

```diff
@@ -1,54 +1,52 @@
-// Font Size
-.textTitle,
-.textH1,
-.textH2,
-.textH3,
-.textH4 {
-  font-weight: map-get($font-weights, bold);
-}
-
-.bodyLargeRegular,
-.bodyLargeSimeBold {
-  font-size: $font-body-large;
-}
-
-.bodySmallRegular,
-.bodySmallSemiBold {
-  font-size: $font-body-small;
-}
+.text {
+  &__title,
```

Comment:
heading (h1.. h2..), already is a title, isn't it?

Reply:
no, the "Title" is an extra large font, designed to display a large text on the screen as shown on the image below, it is represented as "Display"

![Screenshot 2022-12-19 at 15 31 21](https://user-images.githubusercontent.com/41298710/208461640-f98d34d7-e561-4ac5-a1da-0259b5f58605.png)